src/main.c

```diff
@@ -1924,6 +1926,9 @@ tokenDefinition_t* getKnownToken() {
             case CHAIN_KIND_TOBALABA:
                 currentToken = (tokenDefinition_t *)PIC(&TOKENS_TOBALABA[i]);
                 break;
```

Comment:
This break statement was missing in your modified fork. I'll add it during the merge.

Reply:
Fixed.